### PR TITLE
Translate fastlane

### LINKED ISSUE #106: Crowdin translate fastlane
I'm not familiar with setting up crowdin but would it be possible to set up translation for the fastlane metadata?

### PR DESCRIPTION
**Pull request by 6543**
close #106
 * add crowdin config (for documentation purpose)
 * add german, spanish fastlane translation
 * update app translation

### CODE REVIEW

**Comment by 6543** [comment]
@helloworld1 new translations, add crowdin config example, ...

File: fastlane/metadata/android/de-DE/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FreeOTP Plus hat FreeOTP, das von RedHat zur Verfügung gestellt wurde, mit der erweitert um:
```
Comment (6543) [inline fastlane/metadata/android/de-DE/full_description.txt:1]:
```suggestion
FreeOTP Plus hat FreeOTP, das von RedHat zur Verfügung gestellt wurde, erweitert um:
```

File: fastlane/metadata/android/de-DE/full_description.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+FreeOTP Plus hat FreeOTP, das von RedHat zur Verfügung gestellt wurde, erweitert um:
```
Comment (jugendhacker) [inline fastlane/metadata/android/de-DE/full_description.txt:1]:
```suggestion
FreeOTP Plus hat das von RedHat zur Verfügung gestellte FreeOTP um folgende Funktionen erweiter:
```
This sounds way more fluent in my opinion (I'm a native German speaker)

Comment (6543) [reply fastlane/metadata/android/de-DE/full_description.txt:1]: @jugendhacker, can you considder to have a look at all german translations in crowdin?

Comment (jugendhacker) [reply fastlane/metadata/android/de-DE/full_description.txt:1]: Yeah I could do it

**Comment by 6543** [comment]
@jugendhacker updated pull with your suggestions & added you as prove-reader in crowdin

**Comment by jugendhacker** [comment]
Yeah I also adjusted the translations for some other things, but now I have finished looking over it.

**Comment by 6543** [comment]
@helloworld1 ready for merge :rocket:

**Review by jugendhacker — APPROVED** [review]

**Comment by helloworld1** [comment]
Great work! Thanks @jugendhacker and @6543

**Pull request merged**